docs/framework/wcf/feature-details/how-to-create-temporary-certificates-for-use-during-development.md

```diff
@@ -22,7 +22,7 @@ When developing a secure service or client using Windows Communication Foundatio
 The following command creates a self-signed certificate with a subject name of "RootCA" in the Current User Personal store.
 
 ```powershell
-PS $rootCert = New-SelfSignedCertificate -CertStoreLocation cert:\CurrentUser\My -DnsName "RootCA" -TextExtension @("1.3.6.1.4.1.311.21.10={text}1.3.6.1.5.5.7.3.1,1.3.6.1.5.5.7.3.2")
+PS $rootcert = New-SelfSignedCertificate -CertStoreLocation Cert:\CurrentUser\My -DnsName "RootCA" -TextExtension @("2.5.29.37={text}1.3.6.1.5.5.7.3.1,1.3.6.1.5.5.7.3.2") -KeyUsage CertSign,DigitalSignature,KeyEncipherment
```

Comment:
* KeyUsage should be `CertSign,CrlSign,DigitalSignature`.
* 2.5.29.37 (and its value) should be removed
* Add `2.5.29.19={text}CA=true` to TextExtension

These recommendations make a root authority look like roots under the CA/Browser Forum Baseline Requirements (v 1.6.3) (https://cabforum.org/wp-content/uploads/CA-Browser-Forum-BR-1.6.3.pdf) (section 7.1.2.1)